docs/README.md

```diff
@@ -1,4 +1,5 @@
 # Developer docs
 
 - [Filesystem](filesystem.md)
+- [Translation Strategy](TRANSLATIONS.MD)
```

Comment:
I'd prefer to leave only the `README.md` capitalised, as shown by having `filesystem.md` in lower case, but if `TRANSLATIONS.md` is convention for this type of file maybe we should leave it as that.

Reply:
I don't think there is a convention, I've seen variations on LOCALIZATION.MD and others, but I think we can take the lead on this. Happy to lowercase it